.github/workflows/cron.yml

```diff
@@ -29,33 +29,33 @@ jobs:
           curl -LO https://github.com/oras-project/oras/releases/download/v1.2.0/oras_1.2.0_linux_amd64.tar.gz
           tar -xvf ./oras_1.2.0_linux_amd64.tar.gz 
 
-      - name: Pull trivy-java-db
-        run: |
-          mkdir -p ./cache/db
-          lowercase_repo=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')
-          ./oras pull "ghcr.io/${lowercase_repo}:${DB_VERSION}"
-          tar -xvf javadb.tar.gz -C ./cache/db
-
       - name: Build the binary
         run: make build
 
-      - name: Crawl indexes
-        run: make db-crawl
+      - name: Clone maven-index repository
+        uses: actions/checkout@v4
+        with:
+          # Always use aquasecurity/maven-index as it's a private repository # and cannot be forked.
+          # Using ${github.repository_owner} would fail in forks as it doesn't exist there.
+          repository: aquasecurity/maven-index
+          path: maven-index
+          token: ${{ secrets.ORG_REPO_TOKEN }}
 
       - name: Build database
-        run: make db-build
+        run: make db-build MAVEN_INDEX_DIR=maven-index
 
       - name: Compress database
         run: make db-compress
 
       - name: Move DB
         run: mv cache/db/javadb.tar.gz .
 
-      - name: Login to Docker Hub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKERHUB_USER }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      # Commented out for fork testing - Docker Hub and ECR credentials not available
+      # - name: Login to Docker Hub
```

Comment:
Don't forget to revert that commented out code back in this file.
And also bellow.